package.json

```diff
@@ -22,18 +22,21 @@
     "@nuxtjs/i18n": "8.0.0-rc.4",
     "@pinia/nuxt": "^0.4.11",
     "@playwright/test": "^1.37.1",
+    "@testing-library/vue": "^7.0.0",
     "@vueuse/core": "^10.4.1",
     "@vueuse/nuxt": "^10.4.1",
     "eslint": "^8.48.0",
     "eslint-plugin-vitest": "^0.3.1",
+    "happy-dom": "^10.11.2",
     "husky": "^8.0.3",
     "nuxt": "^3.7.1",
+    "nuxt-vitest": "^0.10.4",
     "nuxt-windicss": "^2.6.1",
     "pinia": "^2.1.6",
-    "vitest": "^0.34.3"
+    "vitest": "^0.33.0"
```

Comment:
Have to downgrade this one due to https://github.com/danielroe/nuxt-vitest/issues/289